baseplate/thrift.go

```diff
@@ -0,0 +1,158 @@
+package baseplate
+
+import (
+	"context"
+	"io"
+	"time"
+
+	"github.com/apache/thrift/lib/go/thrift"
+	"github.com/getsentry/raven-go"
+	"github.com/reddit/baseplate.go/edgecontext"
+	"github.com/reddit/baseplate.go/log"
+	"github.com/reddit/baseplate.go/metricsbp"
+	"github.com/reddit/baseplate.go/secrets"
+	"github.com/reddit/baseplate.go/thriftbp"
+	"github.com/reddit/baseplate.go/tracing"
+)
+
+type baseplateThriftServer struct {
+	thriftServer *thrift.TSimpleServer
+	config       ServerConfig
+	afterStop    []io.Closer
```

Comment:
🔕 now without `beforeStop` maybe this should just be called `closers`? I don't feel strongly, though.

Reply:
Actually looking at the `append` calls in constructor I do think the name `closers` makes more sense.